pkg/ebpf/tracee.go

```diff
@@ -1691,6 +1677,7 @@ func (t *Tracee) triggerMemDump(event trace.Event) []error {
 
 	errs := []error{}
 
+	// TODO: consider to iterate over given policies when policies are changed
```

Comment:
It seems to me you for now didn't change `t.config.Policies` for the initialization, and are planning to do this change later, you only using `policy.Snapshots()` in dynamic places for this pr, which is okay for me, though, I think the methods to `Subscribe`, `Unsubscribe`, etc from the GRPC API aren't in the initialization category, and must be updated https://github.com/aquasecurity/tracee/blob/main/pkg/ebpf/tracee.go#L1857-L1946

Reply:
No, this won't break Subscribe since we're not changing policies in userland yet and the assignment of the Policies is done in:

https://github.com/aquasecurity/tracee/pull/3305/files#diff-93e56b073f4afff2e526135681adfab063a5ef036eb2735daa066b9bb9ec242bR234-R235